actionview/test/template/dependency_tracker_test.rb

```diff
@@ -60,11 +60,10 @@ def test_dependency_of_erb_template_with_number_in_filename
   end
 
   def test_dependency_of_template_partial_with_layout
-    skip # FIXME: Needs to be fixed properly, right now we can only match one dependency per line. Need multiple!
     template = FakeTemplate.new("<%# render partial: 'messages/show', layout: 'messages/layout' %>", :erb)
     tracker = make_tracker("multiple/_dependencies", template)
 
-    assert_equal ["messages/layout", "messages/show"], tracker.dependencies
+    assert_equal ["messages/show", "messages/layout"], tracker.dependencies
```

Comment:
Does this test fail with your change or are you just changing the test to match your change? If it's the former - I think both ways would need to pass. If it's the latter let's leave the original test as is and create a second test. Let me know if I am not understanding the issue as it's very early where I am :wink: :coffee: